src/main/resources/db/changesets/20200227_initial_schema.yaml

```diff
@@ -103,6 +103,8 @@ databaseChangeLog:
               name: workspace_id
               type: text
               constraints:
+                references: workspace(workspace_id)
```

Comment:
just some adjustments to the schema i noticed. i can break these out into a separate changeset if we're at a point where we want to not blow up dev DBs

Reply:
It should still be ok to edit these, but yeah it's getting close to the point where we'll need to be careful.